app/jobs/add_inbox_tag_to_threads_job.rb

```diff
@@ -0,0 +1,5 @@
+class AddInboxTagToThreadsJob < ApplicationJob
+  def perform
+    MessageThread.find_each { |message_thread| AddInboxTagToThreadJob.perform_later(message_thread) }
+  end
+end
```

Comment:
Tento job, aj ten dalsi, ktory sa scheduluje v tomto jobe su iba docasne, aby sme mali ako vykonat hromadne update na uz existujucich vlaknach (kedze v migracii by to cele nezbehlo). Potom zmazeme.